flink-connector-hbase-2.2/src/test/java/org/apache/flink/connector/hbase2/HBaseConnectorITCase.java

```diff
@@ -40,18 +40,15 @@
 import org.apache.flink.types.RowKind;
 import org.apache.flink.util.CollectionUtil;
 
-import org.apache.flink.shaded.guava30.com.google.common.collect.Lists;
-
 import org.apache.hadoop.hbase.TableName;
 import org.apache.hadoop.hbase.util.Bytes;
 import org.junit.ClassRule;
 import org.junit.Test;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
looks like this is an issue from Checkstyle's point of view, it doesn't like stars in import

Reply:
Sometimes I hate IDEs... thanks :)

Reply:
this star import could be disabled in IntellijIDEA btw
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class